data/cvr/samples/jetsons/jetsons_bedrock-precinct_cvr.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CastVoteRecordReport xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://itl.nist.gov/ns/voting/1500-103/v1 ../../../../../CastVoteRecords/NIST_V0_cast_vote_records.xsd"
+ xmlns="http://itl.nist.gov/ns/voting/1500-103/v1">
+    <CVR>
+        <BallotImage>
+            <Image FileName="bedrock_ballot_1" MimeType="application/pdf" />
+        </BallotImage>
+        <BallotStyleId>bedrock-precinct-ballot</BallotStyleId>
+        <CurrentSnapshotId>sn_0</CurrentSnapshotId>
+        
+
+        <!-- Base snapshot is derived from the output from the mobile app -->
+        <CVRSnapshot ObjectId="sn_0">
```

Comment:
Use: `snapshot-01`. It's probably not likely there will be more than 9 snapshots but it's not impossible either.

Reply:
done